Sources_v3/StreamChatUI/ChatChannelList/ChatChannelListCollectionViewCell.swift

```diff
@@ -5,4 +5,36 @@
 import Foundation
 import UIKit
 
-open class ChatChannelListCollectionViewCell: UICollectionViewCell {}
+open class ChatChannelListCollectionViewCell<ExtraData: UIExtraDataTypes>: UICollectionViewCell {
+    
+    // MARK: - Properties
+    
+    var uiConfig: UIConfig<ExtraData> = .default
```

Comment:
This is not correct. The config must be passed from the super view.

Reply:
The collection view instantiates the registered cells by itself and returns non-optinal cell from `dequeueReusableCell`. I didn't find the way how to invoke our custom `init(config: UIConfig)` to create a cell instance. 

We have task in which scope the further investigation makes sense: [CIS-423](https://stream-io.atlassian.net/secure/RapidBoard.jspa?rapidView=61&projectKey=CIS&modal=detail&selectedIssue=CIS-423)